.github/workflows/build-test.yml

```diff
@@ -219,3 +219,81 @@ jobs:
       - name: Codecov
         run: 'bash <(curl -s https://codecov.io/bash)'
         if: matrix.python-version == '3.9'
+
+  Linux-ROOT:
```

Comment:
`check-yaml` fails here:
```
 Check failure on line 223 in .github/workflows/build-test.yml

 GitHub Actions
/ .github/workflows/build-test.yml

Invalid workflow file
You have an error in your yaml syntax on line 223
```

Reply:
To get quicker turn-around on YAML formatting issues, I try loading them in [PyYAML](https://pypi.org/project/PyYAML/) to see if there are any errors.

```python
>>> import yaml
>>> ok = yaml.safe_load(open(".github/workflows/build-test.yml"))
```

It's not usually necessary to look at the value of `ok`, though that can help diagnose a correct-syntax-but-wrong-nesting or other semantic issue.

YAML has some nasty corner-cases, such as `on`/`off` and `yes`/`no` mean `true`/`false`, not the strings `"on"`/`"off"` and `"yes"`/`"no"` (even as keys of a dict).

```python
>>> yaml.safe_load("""
... on: 1
... off: 2
... yes: 3
... no: 4
... true: 5
... false: 6
... """)
{True: 5, False: 6}
```

When in doubt, quote it.

The use of "`safe_yaml`" is because YAML, as a format, is allowed to run arbitrary code. We know that our own YAML is not malicious, but I don't even remember the command for the non-safe loader.

Reply:
`yaml.full_load` is an alias for `yaml.load(..., loader=yaml.FullLoader)` IIRC